cmd/ddev/cmd/commands.go

```diff
@@ -542,3 +559,14 @@ func findDirectivesInScriptCommand(script string) map[string]string {
 
 	return directives
 }
+
+func runMutagenSync(app *ddevapp.DdevApp, mutagenSync bool) {
+	if mutagenSync {
```

Comment:
Here or somewhere needs to check to see if mutagen is enabled and the current project is running.

Reply:
`app.MutagenSyncFlush()` already checks for enabled/disabled Mutagen, but it doesn't harm to add it here as well.